pages/SignIn/index.tsx

```diff
@@ -7,85 +7,108 @@ import google from "@/public/google.png";
-import emailCheck from "@/lib/emailCheck";
-import axios from "@/lib/axios";
+import emailCheck from "@/utils/isEmail";
+import axios from "@/api/axios";
 import AccessTokenChecker from "@/components/AccessTokenChecker/AccessTokenChecker";
 
 function SignIn() {
-  const [emailInputValue, setEmailInputValue] = useState("");
-  const [passwordInputValue, setPasswordInputValue] = useState("");
-  const [emailError, setEmailError] = useState("");
-  const [passwordError, setPasswordError] = useState("");
+  const [inputValues, setInputValues] = useState({
+    email: "",
+    password: "",
+  });
+
+  const [errors, setErrors] = useState({
+    emailError: "",
+    passwordError: "",
+  });
 
   const router = useRouter();
 
   const handleEmailInputValueChange = (e: ChangeEvent<HTMLInputElement>) => {
-    setEmailInputValue(e.currentTarget.value);
+    setInputValues(prev => ({
+      ...prev,
+      email: e.target.value,
+    }));
   };
 
   const handlePasswordInputValueChange = (e: ChangeEvent<HTMLInputElement>) => {
-    setPasswordInputValue(e.currentTarget.value);
+    setInputValues(prev => ({ ...prev, password: e.target.value }));
   };
 
   const handleEmailBlur = (e: ChangeEvent<HTMLInputElement>) => {
     if (!e.target.value) {
-      setEmailError("이메일을 입력해 주세요.");
-    } else if (!emailCheck(e.target.value)) {
-      setEmailError("올바른 이메일 주소가 아닙니다.");
-    } else {
-      setEmailError("");
+      setErrors(prev => ({
+        ...prev,
+        emailError: "이메일을 입력해 주세요.",
+      }));
+      return;
+    }
+    if (!emailCheck(e.target.value)) {
+      setErrors(prev => ({
+        ...prev,
+        emailError: "올바른 이메일 주소가 아닙니다.",
+      }));
+      return;
     }
+    setErrors(prev => ({
+      ...prev,
+      emailError: "",
+    }));
   };
   const handlePasswordBlur = (e: ChangeEvent<HTMLInputElement>) => {
     if (!e.target.value) {
-      setPasswordError("비밀번호를 입력해 주세요.");
-    } else {
-      setPasswordError("");
+      setErrors(prev => ({
+        ...prev,
+        passwordError: "비밀번호를 입력해 주세요.",
+      }));
+      return;
     }
+    setErrors(prev => ({
+      ...prev,
+      passwordError: "",
+    }));
   };
 
   async function handleFormSubmit(e: FormEvent<HTMLFormElement>) {
     try {
       e.preventDefault();
-      const response = await axios.post("/api/sign-in", {
-        email: emailInputValue,
-        password: passwordInputValue,
+      const response = await axios.post("sign-in", {
+        email: inputValues.email,
+        password: inputValues.password,
       });
       localStorage.setItem("accessToken", response.data.data.accessToken);
-      router.push("/Folder");
+      router.push("/folder");
     } catch (error) {
-      setEmailError("이메일을 확인해 주세요.");
-      setPasswordError("비밀번호를 확인해 주세요.");
+      setErrors(prev => ({
+        ...prev,
+        emailError: "이메일을 확인해 주세요.",
+      }));
+      setErrors(prev => ({
+        ...prev,
+        passwordError: "비밀번호를 확인해 주세요.",
+      }));
```

Comment:
이런식으로 표현해줘도 괜찮을 것 같네요!
```suggestion
      setErrors({
        passwordError: "비밀번호를 확인해 주세요.",
        emailError: "이메일을 확인해 주세요.",
      });
```